### PR TITLE
Feat/masquerade-post

### PR DESCRIPTION
**Pull request by stdavis**
Also add a few links to it in other pages.

### CODE REVIEW

**Comment by stdavis** [comment]
@steveoh I'd love a review on this when you have a minute.

File: _posts/2022-07-27-introducing-masquerade.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: 'Introducing Masquerade'
+author:
+  display_name: Scott Davis
+  email: stdavis@utah.gov
+date: 2022-07-27 16:20:21
```
Comment (steveoh) [inline _posts/2022-07-27-introducing-masquerade.md:6]:
```suggestion
date: 2022-08-01 12:20:21
```

**Review by steveoh — APPROVED** [review]
The post seems super short. What you wrote on the readme is great. I think it would be nice to duplicate it?

**Comment by stdavis** [comment]
> I think it would be nice to duplicate it?

I can do that...

**Review by steveoh — APPROVED** [review]

**Pull request merged**